src/transformers/models/clip/modeling_clip.py

```diff
@@ -953,15 +958,15 @@ def get_image_features(
     @replace_return_docstrings(output_type=CLIPOutput, config_class=CLIPConfig)
     def forward(
         self,
-        input_ids=None,
-        pixel_values=None,
-        attention_mask=None,
-        position_ids=None,
-        return_loss=None,
-        output_attentions=None,
-        output_hidden_states=None,
-        return_dict=None,
-    ):
+        input_ids: Optional[torch.LongTensor] = None,
+        pixel_values: Optional[torch.FloatTensor] = None,
+        attention_mask: Optional[torch.Tensor] = None,
+        position_ids: Optional[torch.LongTensor] = None,
+        return_loss: Optional[bool] = None,
+        output_attentions: Optional[bool] = None,
+        output_hidden_states: Optional[bool] = None,
+        return_dict: Optional[bool] = None,
+    ) -> Union[Tuple, CLIPOutput]:
```

Comment:
```suggestion
    ) -> Union[Tuple, CLIPOutput]:
```

Fixes committed. I am unsure on this one, as I assume that this returns a different output type than ClipOutput instance

Reply:
awesome - thanks for your help! Little bit of a learning curve..

Tests passed, and make fixup called.